docs/en/advanced_guides/conventions.md

```diff
@@ -2,6 +2,38 @@
 
 Please check the following conventions if you would like to modify MMDetection as your own project.
 
+## Image pipeline input shape
+
+In OpenMMLab 2.0, the image pipeline input shape is `(width, height)`, the shape of the pipeline output is `(height, width)` and the shape is also always `(height, width)` in model. The common fields are as follows:
```

Comment:
In OpenMMLab 2.0, to be consistent with the input argument of OpenCV, the argument about image shape in the data transformation pipeline is always in the `(width, height)` order. On the contrary, for computation convenience, the order of the field go through the data pipeline and model is `(height, width)`. The common fields are as below: